runnables/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM sqlflow/sqlflow:step
+
+RUN apt-get clean && apt-get update && \
+    apt-get -qq install python3-pip libmysqld-dev libmysqlclient-dev && \
+    ln -s /usr/bin/pip3 /usr/bin/pip
+
+ADD ./requirements.txt /
```

Comment:
Put `pip install ...` in the `RUN` line and remove `requirements.txt` to reduce one Docker image layer.

Reply:
When we develop and debug the code for Runnable with VSCode, we can install the dependencies on our Mac using the requirement.txt. 
It'll be more convenient considering the development experience.

Reply:
I see.